doc/nrf/libraries/bluetooth_services/scan.rst

```diff
@@ -8,35 +9,39 @@ Scanning module
    :depth: 2
 
 The scanning module handles the Bluetooth® Low Energy scanning for your application.
-You can use it to find advertising devices and establish connections with them.
 
-Using :ref:`filters <nrf_bt_scan_readme_filters>`, you can narrow down the scan to devices of a specific type.
+Overview
+********
 
-The Scanning Module can work in one of the following modes:
+Use the scanning module to find advertising devices and establish connections with them.
+The module can automatically establish a connection after a filter match.
+It can also receive :ref:`directed advertising packets <lib_nrf_bt_scan_readme_directedadvertising>`.
+
+The scanning module can work in one of the following modes:
 
 * Simple mode - without using filters
 * Advanced mode - that allows using advanced filters
 
-The module can automatically establish a connection after a filter match.
-It can also receive :ref:`directed advertising packets <nrf_bt_scan_readme_directedadvertising>`.
+Configuration
+*************
 
-Usage
-*****
+To initialize the module, call the function :c:func:`bt_scan_init`.
 
+You can also call the function without an initialization structure.
+When you pass the initialization structure as ``NULL``, the default static configuration is used.
 
-You can use the scanning module to execute the following actions:
+This configuration is also used when you use an initialization structure with ``NULL`` pointers to scan parameters and connection parameters.
```

Comment:
```suggestion
This configuration is also used when you initialize a structure with ``NULL`` pointers to scan parameters and connection parameters.
```

Reply:
@ppryga-nordic

Reply:
@grochu Could you please advise on that matter?

Reply:
I prefer the "initialization structure" which effectively is the structure passed to the initialization function: `bt_scan_ini()`.
Or the text could say: "This configuration is also used when you initialize the module with a structure that has ``NULL`` pointers set for scan and connection parameters." or something similar.